python/paddle/v2/framework/initializer.py

```diff
@@ -114,20 +114,20 @@ class NormalInitializer(Initializer):
     """Implements the  random Normal(Gaussian) distribution initializer
     """
 
-    def __init__(self, loc=0.0, scale=1.0, seed=0):
```

Comment:
We should keep these names to loc and scale. They have been taken so that the naming convention is consistent with other frameworks such as numpy and tensorflow.
refer to these: 
http://www.itl.nist.gov/div898/handbook/eda/section3/eda364.htm
https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.random.normal.html#numpy.random.normal